install/kubernetes/helm/istio/charts/tracing/templates/service.yaml

```diff
@@ -39,4 +39,7 @@ items:
     selector:
       app: jaeger
```

Comment:
Not sure how this got in - but please remove this ASAP. No load balancers for internal services, use gateways and only if the app is protected ( auth, mtls, etc).

Reply:
+1, will update this PR by removing lb service type.